src/ext/java/org/opentripplanner/ext/restapi/resources/RequestToPreferencesMapper.java

```diff
@@ -95,6 +96,27 @@ private void mapBike() {
     });
   }
 
+  private void mapScooter() {
+    preferences.withScooter(scooter -> {
+      setIfNotNull(req.bikeSpeed, scooter::withSpeed);
+      setIfNotNull(req.bikeReluctance, scooter::withReluctance);
+      setIfNotNull(
+        req.bikeOptimizeType,
+        optimizeType -> scooter.withOptimizeType(LegacyVehicleRoutingOptimizeType.map(optimizeType))
+      );
```

Comment:
Just so I understand it correctly: until we have these preferences in the API, we will continue using the bike ones for scootering, correct?